packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -314,7 +312,9 @@ export default function vercelServerless({
 							? [
 									{
 										src: '/.*',
-										dest: fourOhFourRoute.prerender ? '/404.html' : 'render',
+										dest: fourOhFourRoute.prerender ? '/404.html'
+											: _middlewareEntryPoint ? '_middleware'
+											: 'render',
```

Comment:
Should this use `dest` instead? Or the `render` should be `_render`

Reply:
I think we should move this strings (`_render`, `_middleware`) to some constants, so we don't do any typo

Reply:
You're right, it is supposed to be `_render`. Moved to constants.